sr_unix/gtmxc_types.h

```diff
@@ -71,6 +71,7 @@ void 		gtm_free(void *);
 void		gtm_hiber_start(ydb_uint_t mssleep);
 void		gtm_hiber_start_wait_any(ydb_uint_t mssleep);
 void		gtm_start_timer(ydb_tid_t tid, ydb_int_t time_to_expir, void (*handler)(), ydb_int_t hdata_len, void *hdata);
+gtm_status_t	gtm_ci_filter(const char *c_rtn_name, ...);
```

Comment:
Looks like GTM added a new externally accessible routine (who does I know not what yet) that likely needs a ydb counterpart. Might be a new utility routine. Depending on what it does, might need a Golang flavor as well if that turns out to be useful.

Reply:
If it is not actually exported, then the declaration should be moved from gtmxc_types.h (which IS user accessible) to some other internal header file.

Reply:
Done. Now moved to sr_unix/gtmci.h.